### PR TITLE
return removeListeners for interaction.addListener

### PR DESCRIPTION
**Pull request by robinlej**

### CODE REVIEW

File: addons/website/static/src/core/colibri.js [hunk 1]
```diff
@@ -82,10 +82,18 @@ export class Colibri {
...
         };
+        const removeListeners = [];
         for (const node of nodes) {
             node.addEventListener(event, handler, options);
-            this.cleanups.push(() => node.removeEventListener(event, handler, options));
+            const removeListener = () => node.removeEventListener(event, handler, options);
+            this.cleanups.push(removeListener);
+            const currIdx = this.cleanups.length - 1;
+            removeListeners.push(() => {
+                this.cleanups.splice(currIdx, 1);
```
Comment (robinlej) [inline addons/website/static/src/core/colibri.js]: maybe not needed, idk

Comment (bso-odoo) [reply addons/website/static/src/core/colibri.js]: `currIdx` will not be stable if this mecanism is used to remove them in a custom sequence

File: addons/website/static/src/core/interaction.js [hunk 2]
```diff
@@ -209,19 +209,23 @@ export class Interaction {
...
-     * cleaned up when the interaction is destroyed
+     * cleaned up when the interaction is destroyed.
+     * Returns a function to remove the listener, or an array of functions if
+     * more than one node was listened to.
      *
      * @param {EventTarget | string} target an element, a bus or a selector
      * @param {string} event
      * @param {Function} fn
      * @param {Object} [options]
+     * @returns {Function[] | Function} removeListeners
```
Comment (bso-odoo) [inline addons/website/static/src/core/interaction.js]: Not true: it returns a single function that cleans them all

File: addons/website/static/src/core/colibri.js (hunk 1 above)
Comment (robinlej) [reply addons/website/static/src/core/colibri.js]: :facepalm: of course

**Pull request merged**